pyats_installer.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
+import subprocess
+import argparse
+import pkg_resources
+from distutils.version import StrictVersion
+LATEST = '19.12'
+
+VERSION_MAPPING = {
+    '20.1': {
+        'install': ['pyats[full]'],
+        'uninstall': ['pyats[full]']
+    },
+    '19.11':{
+        'install': ['unicon', 'unicon.plugins'],
+        'uninstall': ['unicon']
+    }
+}
+# todo find a way to automatically find pkg deps+extras
+# pkg_resource.require doesnt work b/c it raise exception if the dep is not installed
+PYATS_PKG_DEPENDENCIES = {
```

Comment:
look into https://github.com/naiquevin/pipdeptree/blob/master/pipdeptree.py

FrozenRequirement is what you need - I think